xonsh/__init__.py

```diff
@@ -1,5 +1,11 @@
 __version__ = '0.5.3'
 
+import sys as _sys
+if '--timings' in _sys.argv:
+    from xonsh.timings import setup_timings
+    setup_timings()
+del _sys
```

Comment:
So, on Windows clock() reports the time since it was first called. I have adapted the code to only call it on windows and assume 0 on other platforms.